config/application.rb

```diff
@@ -79,6 +85,8 @@ class Application < Rails::Application
     # config.paths.add File.join('app', 'api'), glob: File.join('**', '*.rb')
     # config.autoload_paths += Dir[Rails.root.join('app', 'api', '*')]
 
+    config.x.use_vips = ENV['MASTODON_USE_LIBVIPS'] == 'true'
```

Comment:
Should we define a `use_libvips` variable above, and use this here and on line 31?